framework/db/QueryBuilder.php

```diff
@@ -1067,22 +1067,7 @@ public function buildInCondition($operator, $operands, &$params)
         }
 
         if ($values instanceof Query) {
-            // sub-query
-            list($sql, $params) = $this->build($values, $params);
-            $column = (array) $column;
```

Comment:
This was a bug, making the `else` part below never to execute. I've removed this cast.